storybook/stories/staking/Undelegate.stories.js

```diff
@@ -4,15 +4,24 @@ import BigNumber from 'bignumber.js';
 import moment from 'moment';
 import { linkTo } from '@storybook/addon-links';
 
+import STAKE_POOLS from '../../../source/renderer/app/config/stakingStakePools.dummy.json';
+import { generateWallet } from '../_support/utils';
 // Screens
-import UndelegateConfirmationDialog from '../../../source/renderer/app/components/staking/delegation-center/UndelegateConfirmationDialog';
-import UndelegateConfirmationResultDialog from '../../../source/renderer/app/components/staking/delegation-center/UndelegateConfirmationResultDialog';
+import UndelegateWalletConfirmationDialog from '../../../source/renderer/app/components/wallet/settings/UndelegateWalletConfirmationDialog';
+import UndelegateWalletConfirmationResultDialog from '../../../source/renderer/app/components/wallet/settings/UndelegateWalletConfirmationResultDialog';
+
+const selectedWallet = generateWallet(
```

Comment:
@yakovkaravelov can you also add a story for hardware wallets. Use knobs to switch `isHardwareWallet` true / false and generate one more wallet with `isHardwareWallet` flag set to `true`

Reply:
👍